doc/ompl_interface_tutorial.rst

```diff
@@ -1,8 +1,34 @@
-OMPL Interface
-========================================
+OMPL Interface Tutorial
+=======================
 
-OMPL Optimization Objective Tutorial
-------------------------------------
+The Open Motion Planning Library is a powerful collection of state-of-the-art sampling-based motion planning algorithms and is the default planner in MoveIt!. For more information see `project webpage <http://ompl.kavrakilab.org/>_`.
+
+OMPL Settings
+-------------
+
+Here we review important configuration settings for OMPL. These settings can typically be found in the ``ompl_planning.yaml`` file located in your robots ``moveit_config`` package.
+
+Longest Valid Segment Fraction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``longest_valid_segement_fraction`` defines the discretization of robot motions used for collision checking, and greatly affects the performance and reliability of OMPL-based solutions. A ``motion`` in this context can be thought of as an edge between two nodes in a graph, where nodes are discretized robot states that randomly sampled and collision checked for validity. The default motion collision checker in OMPL simply discretizes the edge into a number of sub-states to collision check. Currently no continuous collision checking is available in OMPL/MoveIt!, though this is an area of current `discussion <https://github.com/ros-planning/moveit/issues/29>_`.
+
+Set ``longest_valid_segement_fraction`` too low, and collision checking / motion planning will be very slow. Set too high and collisions will be missed around small or narrow objects. In addition, a high collision checking resolution will cause the path smoothers to output incomprehsible motions because they are able to "catch" the invalid path and then attempt to repair them by sampling around it, but imperfectly.
+
+A quick analysis of the affect of this parameter on two of the MoveIt! tutorial examples is documented `here <https://github.com/ros-planning/moveit/pull/337>_`.
```

Comment:
> how is your English so much better than mine? :us: ;-)

I'm rather sure it's not. I just write shorter texts ;)

> Github is here to stay

Yeah, sadly it's unavailable now and then and (much more importantly) not everyone works connected to the internet the whole time, so I still believe it's best to keep documentation together in one place instead of using links as documentation.

But I know this is a bit of work and it's great that your spending time with the docs at all :)